packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -150,26 +176,50 @@ class HorizontalOverflow extends Foundation<
     }
 
     /**
-     * Callback on scroll change
+     * Callback for on scroll change
      */
     private onScrollChange = (): void => {
-        if (!this.props.onScrollChange) {
-            return;
+        if (typeof this.props.onScrollChange === "function") {
+            this.props.onScrollChange(this.getScrollChangeData());
         }
+    };
 
+    /**
+     * Observe resize of parent element
+     */
+    private onResizeObserver = (): void => {
```

Comment:
This should not be its own method, I would instead place this in componentDidMount as once the component is mounted you will have access to the ref.